cmd/node-cache/app/cache_app.go

```diff
@@ -93,6 +94,12 @@ func (c *CacheApp) Init() {
 	c.params.SetupIptables = setupIptables
 }
 
+// isIPv6 return if the node-cache is working in IPv6 mode
+// LocalIPs are guaranteed to have the same family
+func (c *CacheApp) isIPv6() bool {
+	return utilnet.IsIPv6(c.params.LocalIPs[0])
```

Comment:
Can we handle the case where no localIPs are specified and return false in that case.

Reply:
sure